static/app/views/integrationPipeline/awsLambdaFunctionSelect.spec.tsx

```diff
@@ -16,6 +16,30 @@ describe('AwsLambdaFunctionSelect', () => {
     );
     expect(screen.getByRole('checkbox', {name: 'lambdaB'})).toBeInTheDocument();
     expect(screen.getByRole('button', {name: 'Finish Setup'})).toBeInTheDocument();
-    // TODO: add assertion for form post
+
+    // Uncheck lambda A
+    await userEvent.click(screen.getByRole('checkbox', {name: 'lambdaA'}));
+
+    const {promise, resolve} = Promise.withResolvers<FormData>();
```

Comment:
would avoid until babel can compile it into something older https://caniuse.com/mdn-javascript_builtins_promise_withresolvers